### PR TITLE
Add missing license headers to files to placate the compliance checks.

### PR DESCRIPTION
**Pull request by usergenic**
g3 sync prereq

### CODE REVIEW

File: packages/labs/virtualizer/src/layouts/shared/SizeCache.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
```
Comment (AndrewJakubowicz) [inline packages/labs/virtualizer/src/layouts/shared/SizeCache.ts:3]: The SizeCache file was added in this commit: https://github.com/PolymerLabs/uni-virtualizer/commit/c1b6c56b517a33f1af2e207cfbab6cb64b96ba58#diff-8f89c300b7e83bcbf8413f790b77a41ab4a91f901b855bd18b7a51e6b79a0196

The copyright year should be the year it was created - thus `2021`.

File: packages/labs/virtualizer/src/layouts/shared/SizeGapPaddingBaseLayout.ts [hunk 2]
```diff
@@ -1,3 +1,9 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
```
Comment (AndrewJakubowicz) [inline packages/labs/virtualizer/src/layouts/shared/SizeGapPaddingBaseLayout.ts:3]: Change to `2021` please - which was when [`SizeGapPaddingBaseLayout`](https://github.com/PolymerLabs/uni-virtualizer/commit/c1b6c56b517a33f1af2e207cfbab6cb64b96ba58#diff-9c8673c6a269190f679f82f4372aa96f33f9bf5acd10621fce4dd3a4b497de0c) was added. Thank you!

File: packages/labs/virtualizer/src/layouts/shared/SizeCache.ts (hunk 1 above)
Comment (usergenic) [reply packages/labs/virtualizer/src/layouts/shared/SizeCache.ts]: Thanks, @AndrewJakubowicz - didn't realize we were to backdate license headers to file creation date.  Seems fair in this case anyways.  Done.

File: packages/labs/virtualizer/src/layouts/shared/SizeGapPaddingBaseLayout.ts (hunk 2 above)
Comment (usergenic) [reply packages/labs/virtualizer/src/layouts/shared/SizeGapPaddingBaseLayout.ts]: 👍

**Review by AndrewJakubowicz — APPROVED** [review]

**Pull request merged**